src/main/java/io/IOHandler.java

```diff
@@ -0,0 +1,8 @@
+package io;
+
+public interface IOHandler {
+    String readInput();
+    void writeOutput(String output);
+    void greet();
+    void sayGoodbye();
+}
```

Comment:
Interesting choice to implement an interface for your ConsoleIO class! But since there aren't likely to be other classes implementing IOHandler interface, you might consider not including this interface for simplicity reasons.